fdbserver/workloads/PhysicalShardMove.actor.cpp

```diff
@@ -489,9 +528,6 @@ struct PhysicalShardMoveWorkLoad : TestWorkload {
 	                                         int teamSize,
 	                                         std::unordered_set<UID> includes,
 	                                         std::unordered_set<UID> excludes) {
```

Comment:
is the comment "after disabling DD, ..." still valid?

Reply:
Revised the comments.